ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -4093,8 +4093,22 @@ let vm_clone_aux clone_op cloned_string printer include_template_vms rpc
 
 let vm_clone printer = vm_clone_aux Client.VM.clone "Cloned " printer true
 
-let vm_snapshot printer =
-  vm_clone_aux Client.VM.snapshot "Snapshotted " printer false
+let vm_snapshot printer rpc session_id params =
+  let ignore_vdis_uuids =
+    match List.assoc_opt "ignore-vdis" params with
+    | None ->
+        []
+    | Some x ->
+        String.split_on_char ',' x
+  in
+  let ignore_vdis =
+    List.map
+      (fun vdi_uuid -> Client.VDI.get_by_uuid rpc session_id vdi_uuid)
```

Comment:
What happens if the parameter contains a UUID that does not belong to a VDI? The `get_by_uuid` would fail - how is this handled?

Reply:
This gives this error to CLI:
```
[11:15 test-brs-b ~]# xe vm-snapshot vm=SVE\ Alpine\ minimal\ for\ exports new-name-label=prime-video ignore-vdis=toto
The uuid you supplied was invalid.
type: VDI
uuid: toto
```
Which I think is the expected behavior, and the same than for other method with an invalid uuid.